src/Components/Footer.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import '../css/Footer.css';
+
+function Footer(props) {
+  const {item,
+        showActive,
+        showCompleted,
+        showAll,
+        clearCompleted,
+        isEmptyTaskBoard,
+        completedTasks,
+        clickOnFooter,
+        handleChangeFooter} = props
+
+const chekStyle = {
+  border: "2px solid #e0e0e0",
+  borderRadius: "5px"
+}
+
+console.log(clickOnFooter);
+
+  const printFooter = ()=>{
+
+    if(isEmptyTaskBoard){
```

Comment:
```suggestion
    if(!isEmptyTaskBoard){
      return null
    }
```

Any time you should wrap your code in one big `if` please check whether you can use `return` right before this code with opposite condition